PostProcessing/Runtime/Effects/MotionBlur.cs

```diff
@@ -58,6 +58,22 @@ public override DepthTextureMode GetCameraFlags()
             return DepthTextureMode.Depth | DepthTextureMode.MotionVectors;
         }
 
+        private void CreateTemporaryRT(PostProcessRenderContext context, int nameID, int width, int height, RenderTextureFormat RTFormat)
+        {
+#if !UNITY_2019_1_OR_NEWER
```

Comment:
Why is this ! UNITY_2019_1_OR_NEWER? Is the logic backwards?

Reply:
The logic isn't backwards but I can make the logic more readable by just removing those lines.